src/librustc_interface/passes.rs

```diff
@@ -159,13 +156,12 @@ pub fn configure_and_expand(
 }
 
 pub struct ExpansionResult {
```

Comment:
Is there a reason to re-wrap the resolve outputs here? I guess we do get Steal...

Reply:
`ExpansionResult` is removed.